### DISCUSSION TITLE
Using existing cluster epiphany and support of GCP cluster

### DISCUSSION THREAD

**Discussion by komalkadam** [epiphany-platform/epiphany#2674] [Q&A]
I went through the documentation of epiphany, wanted to understand the few things whether and how those can be supported
1. It doesn't support using the existing cluster. Just wanted to confirm if there is a way to use the existing cluster?
2. Is there support for the GKE?
3. How can we add support for MangoDB or any new components?

**Comment by rafzei** [comment]
:raised_hand_with_fingers_splayed: Hello @komalkadam, thanks for the questions.
For now, creating infra on GCP is not supported. In case you would like to use Epiphany on top of existing infrastructure please refer to [doc](https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/howto/CLUSTER.md#how-to-create-an-epiphany-cluster-on-existing-infrastructure)
There is no MongoDB on our roadmap, as you are the first person who asked for it. 
Please, tell us more about the MongoDB use case and/or about your project so we can discuss it with the team.

**Reply by komalkadam** [reply]
@rafzei Thanks for the response. Can we use the existing created EKS or AKS cluster to deploy all the components? 
I see that it can use existing infrastructure. But I don't see an option to use existing cluster
